lib/src/components/NxTransferList/TransferListHalf.tsx

```diff
@@ -27,11 +39,36 @@ function TransferListItem<T extends string | number = string>(props: TransferLis
 
   return (
     <NxOverflowTooltip>
-      <label className="nx-transfer-list__item">
-        <NxFontAwesomeIcon icon={checked ? faTimesCircle : faPlusCircle} />
-        <input className="nx-transfer-list__checkbox" type="checkbox" checked={checked} onChange={onChange} />
-        <span>{displayName}</span>
-      </label>
+      <div className={classnames('nx-transfer-list__item',
+          { 'nx-transfer-list__item--with-reordering': !!showReorderingButtons })}>
+        <label className="nx-transfer-list__select">
+          <NxFontAwesomeIcon icon={checked ? faTimesCircle : faPlusCircle} />
+          <input className="nx-transfer-list__checkbox" type="checkbox" checked={checked} onChange={onChange} />
+          <span>{displayName}</span>
+        </label>
+        { showReorderingButtons && (
+          <NxButtonBar className="nx-transfer-list__button-bar">
+            <NxButton variant="icon-only"
+                      className={classnames('nx-transfer-list__button',
```

Comment:
Also pull out these `classnames` calls into local vars

Reply:
> Also pull out these `classnames` calls into local vars

Moved it to its own variable here: 774e9702b6a1123bac0705354974edd6087775a5